/**:
  ros__parameters:
    detection_area:
      stop_margin: 0.0
